src/lib/glob-auto-lib.ts

```diff
@@ -46,7 +46,7 @@ export async function checkerRun(selector: string, lang: string, checks: GalChec
   try {
     let issues: ICreateIssue[] = [];
 
-    await element.all(by.xpath(selector + "//*[not(descendant::div)]")).each(async elem => {
```

Comment:
Please add a check for nested elements - e.g. the "<div>div><div><span>Hardcode</span></div></div></div>" should be reported only once.

Reply:
Added test case to check "<div><table><tr><td><span>It's English text</span></td></tr></table></div>"
With old selector, it will check 5 elements  and report 5 hard code issues. 
With new selector, only one element will be checked and report 1 issue.